style.css

```diff
@@ -130,8 +133,18 @@ body {
 	border-bottom: 1px solid var(--dark-border-color);
 }
 
-
 /* Centers some stuff*/
 .ng-table table th{
     vertical-align: middle;
 }
+
+/* Scroll buttons style*/
+.scroll-btns-grp  {
+  position: fixed; /* Fixed position */
```

Comment:
You can use the CSS class from Bootstrap instead 👍 
https://getbootstrap.com/docs/4.3/utilities/position/#common-values

Reply:
Changed